front/src/applications/operationalStudies/views/v2/ManageTrainScheduleV2.tsx

```diff
@@ -61,6 +66,16 @@ const ManageTrainScheduleV2 = () => {
     () => getPathVoltages(pathProperties?.electrifications, pathProperties?.length),
     [pathProperties]
   );
+  // TODO TS2 : test this hook in simulation results issue
+  if (trainIdToEdit) {
+    useSetupItineraryForTrainUpdate(setPathProperties, trainIdToEdit);
+  }
+
+  // const { data: pathWithElectrifications = { electrification_ranges: [] as RangedValue[] } } =
+  //   osrdEditoastApi.endpoints.getPathfindingByPathfindingIdElectrifications.useQuery(
+  //     { pathfindingId: pathFindingID as number },
+  //     { skip: !pathFindingID }
+  //   );
```

Comment:
Remnant from a debugging session? :P 

Or is this for a future feature?

Reply:
Yep it should be removed